### PR TITLE
Basic Theme

### PR DESCRIPTION
**Pull request by adamhstewart**
Just getting started...

image

image

image

### CODE REVIEW

**Comment by adamhstewart** [comment]
@amitaibu should we add `public/build` to `.gitignore`?

**Comment by amitaibu** [comment]
:+1: 

> should we add public/build to .gitignore?

No, I've deliberately removed it, so on Termux we won't have to `yarn build` - and just serve existing compiled.

**Comment by amitaibu** [comment]
You can `yarn encore dev --watch` - and before merge we'd change it to `yarn encore production`

**Comment by adamhstewart** [comment]
[PureCSS](https://github.com/pure-css/pure) for buttons, forms and tables gives a nice default.

Hello_ChildController_

**Comment by adamhstewart** [comment]
A nice app navigation:
Hello_ChildController_

**Comment by adamhstewart** [comment]
Styling app lists:

Hello_GroupMeetingController_

**Comment by adamhstewart** [comment]
Some form styling:

image

**Comment by adamhstewart** [comment]
@amitaibu I think I've taken this far enough - let me know if you want any different directions.

**Comment by amitaibu** [comment]
https://github.com/amitaibu/mdr-git/pull/11#issue-357576615

File: app/templates/child/show.html.twig [hunk 1]
```diff
@@ -1,68 +1,98 @@
...
-        </div>
+
+      <h3>Add New Measurements</h3>
+
     {% endif %}
 
-    <div>
-        {{ form(form) }}
+    <div class="app-form">
+      {% form_theme form 'forms/form-rows.html.twig' %}
```
Comment (amitaibu) [inline app/templates/child/show.html.twig]: :+1:

File: app/templates/forms/form-rows.html.twig [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    <div class="form-item">
+      {{ form_label(form) }}
+      {{ form_errors(form) }}
+      {{ form_widget(form) }}
+    </div>
+
+{% endblock %}
+
+{% block _child_measurements_height_row %}
```
Comment (amitaibu) [inline app/templates/forms/form-rows.html.twig]: Nice. This is overriding by naming convention?

Comment (adamhstewart) [reply app/templates/forms/form-rows.html.twig]: yeah, it's `_[ID of block]_row` (for overriding the form_row template). You can explicitly set the block ID when you create the form as well.

**Comment by amitaibu** [comment]
Checked on my Android phone, if it won't require too much effort, maybe worth making it look bigger on small screens.

Screenshot_20191228-214846
Screenshot_20191228-214912

**Comment by adamhstewart** [comment]
Child

**Comment by adamhstewart** [comment]
@amitaibu ^^ that's the easiest way to make everything bigger. Everything is in `ems` so it should work pretty well to slide that up or down.

**Comment by amitaibu** [comment]
:100:

**Pull request merged**